lib/codegen/CreateTerm.cpp

```diff
@@ -907,7 +916,7 @@ CreateTerm::createAllocation(KOREPattern *pattern, std::string locationStack) {
                               ->getCategory(Definition)
                               .cat;
                symbolDecl->getAttributes().count("sortInjection")
-               && (cat == SortCategory::Symbol || cat == SortCategory::Bytes)) {
+               && (cat == SortCategory::Symbol)) {
```

Comment:
Resolved a conflict here. Kept the version from master but just dropped `|| cat = SortCategory::Bytes`. The conditional is then equivalent to the pre-Unicode-changes commit, just slightly restructured.
```
<<<<<<< HEAD                                                                                                                                                                                                                                  
    } else if (auto cat = dynamic_cast<KORECompositeSort *>(                                                                                                                                                                                  
                              symbol->getArguments()[0].get())                                                                                                                                                                                
                              ->getCategory(Definition)                                                                                                                                                                                       
                              .cat;                                                                                                                                                                                                           
               symbolDecl->getAttributes().count("sortInjection")                                                                                                                                                                             
               && (cat == SortCategory::Symbol || cat == SortCategory::Bytes)) {                                                                                                                                                              
      std::pair<llvm::Value *, bool> val = createAllocation(                                                                                                                                                                                  
          constructor->getArguments()[0].get(),                                                                                                                                                                                               
          fmt::format("{}:0", locationStack));                                                                                                                                                                                                
=======                                                                                                                                                                                                                                       
    } else if (                                                                                                                                                                                                                               
        symbolDecl->getAttributes().count("sortInjection")                                                                                                                                                                                    
        && dynamic_cast<KORECompositeSort *>(symbol->getArguments()[0].get())                                                                                                                                                                 
                   ->getCategory(Definition)                                                                                                                                                                                                  
                   .cat                                                                                                                                                                                                                       
               == SortCategory::Symbol) {                                                                                                                                                                                                     
      std::pair<llvm::Value *, bool> val                                                                                                                                                                                                      
          = createAllocation(constructor->getArguments()[0].get());                                                                                                                                                                           
>>>>>>> parent of 4c9d010 (Improve handling of Unicode vs Byte strings (#828)) 
```